CONTRIBUTING.md

```diff
@@ -13,6 +13,10 @@ This document describes how to contribute to Batarang
 
 ## Running the tests
 
+## Creating a PR
+
+Please follow the code & commit guidelines found in [Angular core's CONTRIBUTING.md](https://github.com/angular/angular.js/blob/master/CONTRIBUTING.md#submitting-a-pull-request).
```

Comment:
Not sure what you mean - this is pointing to master on the Angular.js repo, not Batarang.